optimize/backend/src/it/java/io/camunda/optimize/service/db/es/report/process/single/flownode/frequency/groupby/flownode/FlowNodeFrequencyByFlowNodeReportEvaluationIT.java.out

```diff
@@ -105,6 +105,86 @@
 //   }
 //
 //   @Test
+//   public void testCollapsedNodesAreExcludedWhenUnsavedReportViewedViewedAsHeatmap()
```

Comment:
We could consider adding a test case for a non heatmap report to ensure this filtering does not happen there, though that wouldnt cover all possible types anywway so I would also be ok not including this.

Reply:
Fair point, I kind of went for the easy option here because of time, but agree that your suggested test case would make sense. I will not do this here (also because of time) unless you feel strongly, but it sounds like this is not the case

Reply:
Should be fine if we cover this during QA imo